src/libs/xinterface/src/QuestFileReader/QuestFileReader.cpp

```diff
@@ -167,7 +167,7 @@ std::string QuestFileReader::GetRecordText(const std::string_view &questID, cons
     const auto &unwrappedText = quest.texts[textNum.value()];
     AssembleStringToBuffer(unwrappedText.str.c_str(), buffer, questData_);
 
-    return std::string(buffer.data(), buffer.size());
+    return buffer;
```

Comment:
This is just an accompanying fix. I've changed vector to string, but forgot about that line. No errors here, just useless conversion.